x-pack/solutions/observability/plugins/observability_onboarding/e2e/playwright/playwright.config.ts

```diff
@@ -72,7 +72,7 @@ export default defineConfig({
       testMatch: '*stateful/*.spec.ts',
       use: {
         ...devices['Desktop Chrome'],
-        viewport: { width: 1920, height: 1200 },
+        viewport: { width: 1920, height: 2400 },
```

Comment:
Firehose onboarding page can get quite long when data is ingested, increasing the height to be able to capture more info on screenshots in case the test fails.